src/test/java/com/commercetools/sync/producttypes/utils/AttributeDefinitionUpdateActionUtilsTest.java

```diff
@@ -495,15 +497,141 @@ public void buildActions_WithSameValues_ShouldReturnEmpty() {
         assertThat(result).isEmpty();
     }
 
+    @Test
```

Comment:
for set of enums we don't have tests for `ChangeEnumValueOrder`, `ChangePlainEnumValueLabel`, `ChangeLocalizedEnumValueOrder`, `ChangeLocalizedEnumValueLabel` update actions.
Maybe we should add this as an issue, wdyt?

Reply:
58abeb9

Reply:
more unit tests: 62922a0
more integration tests: 6080fab

Reply:
thanks for the catch ;)